docs/updating/migration-to-cdn/angular.md

```diff
@@ -0,0 +1,177 @@
+
+@Component( {
+	selector: 'app-root',
+	templateUrl: './app.component.html',
+	styleUrls: ['./app.component.css'],
+	encapsulation: ViewEncapsulation.None,
+	imports: [ CKEditorModule ],
+	standalone: true
+} )
+export class AppComponent {
+	title = 'angular';
+
+	public Editor = ClassicEditor;
+	public config = {
+		licenseKey: '<YOUR_LICENSE_KEY>', // Or 'GPL'.
+		plugins: [ Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo ],
+		toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ],
+		// ... Other configuration ....
+	}
+}
+```
+
+**After:**
+
+```ts
+// app.component.ts
+
+import { Component, ViewEncapsulation } from '@angular/core';
+import { CKEditorModule } from '@ckeditor/ckeditor5-angular';
+
+@Component( {
+	selector: 'app-root',
+	templateUrl: './app.component.html',
+	styleUrls: ['./app.component.css'],
+	encapsulation: ViewEncapsulation.None,
+	imports: [ CKEditorModule ],
+	standalone: true
+} )
+export class AppComponent {
+	title = 'angular';
+
+	public Editor;
+	public config;
+
+	public ngOnInit() {
+		// ADDED
+		loadCKEditorCloud( {
+			version: '43.0.0'
+		} )
+			.then( this._setupEditor.bind( this ) );
+	}
+
+	private _setupEditor( cloud ) {
+		const {
+			ClassicEditor, Bold, Essentials, Italic,
+			Mention, Paragraph, SlashCommand, Undo
+		} = cloud.CKEditor;
+
+		this.Editor = ClassicEditor;
+		this.config = {
+			licenseKey: '<YOUR_LICENSE_KEY>', // Or 'GPL'.
+			plugins: [ Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo ],
+			toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ],
+			// ... Other configuration ....
+		}
+	}
+}
+```
+
+```html
+// app.component.html
+
+<ckeditor
+	*ngIf="Editor"
+	[editor]="Editor"
+	[config]="config"
+	...
+</ckeditor>
+```
+
+### Step 3 (Optional): Migrate the CKEditor&nbsp;5 Angular integration testing suite
+
+If you have any tests that use CKEditor&nbsp;5 objects, you need to update them to use the  `loadCKEditorCloud` function. Here is an example of migrating a test that uses the `ClassicEditor` object:
+
+**Before:**
+
+```javascript
+import { ClassicEditor, ... } from 'ckeditor5';
+
+it( 'ClassicEditor test', () => {
+	// Your test that uses the CKEditor 5 object.
+} );
+```
+
+**After:**
+
+```javascript
+import { loadCKEditorCloud } from '@ckeditor/ckeditor5-angular';
+
+let cloud;
+
+beforeEach( async () => {
+	cloud = await loadCKEditorCloud( {
+		version: '{@var ckeditor5-version}',
+	} );
+} );
+
+it( 'ClassicEditor test', () => {
+	const { ClassicEditor, ... } = cloud.CKEditor;
+
+	// Your test that uses the CKEditor 5 object.
+} );
+```
+
+### Step 4 (Optional): Clean up the document head entries before each test
+
+If you are using a testing suite that does not clean up the document head entries before each test, you may need to do it manually. This is important because the CKEditor&nbsp;5 CDN script will inject the editor into the head section of your HTML file and you need to ensure that the head section is clean before each test.
```

Comment:
```suggestion
If you use a testing suite that does not clean up the document head entries before each test, you may need to do it manually. It is essential because the CKEditor&nbsp;5 CDN script will inject the editor into the head section of your HTML file and, you need to ensure that the head section is clean before each test.
```